test/integration/integration-test.js

```diff
@@ -536,16 +545,48 @@ describe('Dynogels Integration Tests', function () {
       });
     });
 
-    it('should fail with custom error', done => {
-      User.update({
-        id: '123456789',
-        custom: 'forbidden'
-      }, (err, acc) => {
-        expect(err).to.exist;
-        expect(acc).to.not.exist;
-        expect(err).to.match(/Custom field is prohibited/);
-        done();
-      });
+    it('should fail on update with custom error', done => {
+      CustomUser.create(
+        {
+          id: '0987654321',
+          email: 'update@test.com'
+        },
+        (err, acc) => {
+          expect(err).to.not.exist;
+          expect(acc).to.exist;
+          CustomUser.update({
+            id: acc.get('id'),
+            custom: 'forbidden'
+          }, (err, acc) => {
+            expect(err).to.exist;
+            expect(err.update).to.exist;
+            expect(acc).to.not.exist;
+            expect(err.update).to.match(/Custom field is prohibited/);
+            done();
+          });
+        });
+    });
+
+    it('should fail when removing a required attribute with a custom error', done => {
+      CustomUser.create(
+        {
+          id: '1234567890',
+          email: 'email@test.com'
+        },
+        (err, acc) => {
+          expect(err).to.not.exist;
+          expect(acc).to.exist;
+          CustomUser.update(
+            { id: acc.get('id'), email: null },
+            (err, acc) => {
+              console.log(err);
```

Comment:
Not sure what the test policy is for having log statements but it seems unnecessary.